test/python/algorithms/gradients/test_qfi.py

```diff
@@ -51,16 +47,13 @@ def test_qfi_simple(self):
         param_list = [[np.pi / 4, 0.1], [np.pi, 0.1], [np.pi / 2, 0.1]]
         correct_values = [[[1, 0], [0, 0.5]], [[1, 0], [0, 0]], [[1, 0], [0, 1]]]
 
-        qfi = LinCombQFI(self.estimator)
+        qfi = QFI(self.qgt)
         for i, param in enumerate(param_list):
-            qfis = qfi.run([qc], [param]).result().qfis
-            np.testing.assert_allclose(qfis[0], correct_values[i], atol=1e-3)
+            qgts = qfi.run([qc], [param]).result().qgts
+            np.testing.assert_allclose(qgts[0], correct_values[i], atol=1e-3)
```

Comment:
Unit tests failing in the CI run:

```suggestion
            qfis = qfi.run([qc], [param]).result().qfis
            np.testing.assert_allclose(qfis[0], correct_values[i], atol=1e-3)
```